website/blog/2023-01-12-version-071.md

```diff
@@ -0,0 +1,246 @@
+
+With flex gap, this layout can be achieved by setting `gap: 10` for a 10px gap between the interior of the cards:
+
+![[Two diagrams. On the left it shows a skeleton of an app with three boxes that have margin only on the inner sides and not the outer sides of the boxes due to the Flexbox gap property. On the right, the same diagram is shown highlighted to demonstrate the margin only on the inner sides.](/blog/assets/FlexboxGapAfter.png)
+
+For more information on Flexbox gaps, see [_https://css-tricks.com/minding-the-gap/#aa-flexbox-gaps_](https://css-tricks.com/minding-the-gap/#aa-flexbox-gaps).
+
+## Web-inspired props for accessibility, styles, and events
+
+:::note
+
+For more background on aligning React Native to web standards, check out this [_proposal_](https://github.com/necolas/discussions-and-proposals/blob/reduce-fragmentation/proposals/0000-reduce-fragmentation.md) and [_related discussion_](https://github.com/react-native-community/discussions-and-proposals/pull/496).
+:::
+
+This release includes a number of new props inspired by web standards to align React Native’s APIs across many platforms. These new props are purely additive so there are no expected migrations or change of behavior for equivalent accessibility, behavior, or style props.
+
+For any new prop alias introduced, if there is an existing prop with a different name and both are specified, the new alias prop value will take precedence. For example, this release adds a `src` prop alias for `source` on the Image component to align with the `src` prop on web. If both `src` and `source` are provided, the new `src` prop will be used.
+
+#### Accessibility
+
+We introduced ARIA props as alias to existing React Native accessibility props.
+
+These props now exist on all core components of React Native:
+
+- `aria-label`
+- `aria-labelledby`
+- `aria-modal`
+- `id`
+- `aria-busy`
+- `aria-checked`
+- `aria-disabled`
+- `aria-expanded`
+- `aria-selected`
+- `aria-valuemax`
+- `aria-valuemin`
+- `aria-valuenow`
+- `aria-valuetext`
+
+We also introduced equivalent web behavior for the following props:
+
+- `aria-hidden`
+- `aria-live`
+- `role`
+- `tabIndex`
+
+See this [_issue_](https://github.com/facebook/react-native/issues/34424) for more details.
+
+#### Component-Specific Behavior
+
+There were also props introduced to align prop names with equivalent DOM prop names for core components.
+
+**Image**
+
+- `alt`
+- `tintColor`
+- `crossOrigin`
+- `height`
+- `referrerPolicy`
+- `src`
+- `srcSet`
+- `width`
+
+**TextInput**
+
+- `autoComplete`
+- `enterKeyHint
+- `inputMode`
+- `readOnly`
+- `rows`
+
+See this [_issue_](https://github.com/facebook/react-native/issues/34424) for more details.
+
+#### Styles
+
+To align with certain CSS styles, there have been feature extensions for the following styles:
+
+- [`aspectRatio`](https://reactnative.dev/docs/next/layout-props#aspectratio) now supports string values
```

Comment:
We link some of the props to the docs added but not all. I think ideally we should link all props mentioned here to their docs.

However, I'm wondering if we should wait until 0.71 docs are cut over, and then add the links. Since `next` will be wrong. So for now, should we remove the links for all these?

Reply:
> However, I'm wondering if we should wait until 0.71 docs are cut over, and then add the links -- since next will be wrong. So for now, should we remove the links for all these?

One workaround would be to just remove the `/next` portion of the link, since once we merge the docs cut PR 0.71 will be current version so the link will all be as such (ex. https://reactnative.dev/docs/height-and-width) and we'll merge first the docs cut, then the blogpost PR. (as in, no need to wait for the 0.71 docs to have rolled out, we can "pre-mock" the urls)